admin/acceptance.py

```diff
@@ -483,6 +529,10 @@ def main(reactor, args, base_path, top_level):
 
     try:
         nodes = yield runner.start_nodes(reactor)
+        if options['distribution'] in ('fedora-20', 'centos-7'):
+            remote_logs_file = open("remote_logs.log", "a")
+            for node in nodes:
+                capture_journal(reactor, node.address, remote_logs_file)
```

Comment:
How do these connections end?  I guess they live for the entire process lifetime and then die when the reactor stops...